beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  title: string;
+  gameTime: string;
+  phase: string;
+  season: string;
+  year: string;
+  gameType: string;
+  playType: string;
+  rank: string;
+}
+
+const WDInfoDisplay: React.FC<WDInfoDisplayProps> = function ({
+  title,
+  gameTime,
+  phase,
+  season,
+  year,
+  gameType,
+  playType,
+  rank,
+}) {
+  const tableCellStyles = {
+    p: "0px 5px 0px 0px",
+    fontSize: "0.7rem",
+    border: "none",
+  };
+
+  return (
+    <TableContainer>
+      <Table
+        sx={{
+          maxWidth: "250px",
+        }}
+        size="small"
+        aria-label="a dense table"
+      >
+        <TableHead>
+          <TableRow>
+            <TableCell
+              sx={{
+                fontWeight: 600,
+                p: "2px 5px 5px 0px",
```

Comment:
NIT: When setting a value to `0` you don't need the `px` so this could be `2px 5px 5px 0`